tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -273,9 +273,7 @@ var _ = Describe("Verify Basic Cluster Creation", Ordered, func() {
 			var expectResult = []string{"client-ca.crt",
 				"client-ca.key",
 				"client-ca.nochain.crt",
-				"client-supervisor.crt",
```

Comment:
Are these files non existing anymore?

Reply:
They are added to the rotation list here https://github.com/rancher/rke2/issues/5652